apps/nowcasting-app/components/charts/remix-line.tsx

```diff
@@ -197,20 +213,68 @@ const RemixLine: React.FC<RemixLineProps> = ({
     return new Date(now).setHours(o, 0, 0, 0);
   });
 
+  //get Y axis boundary
+
+  const yMaxZoom_Levels = [
+    10, 20, 30, 40, 50, 60, 70, 80, 90, 100, 200, 300, 400, 500, 1000, 2000, 3000, 4000, 5000, 6000,
+    7000, 8000, 9000, 10000, 11000, 12000
+  ];
+
+  let zoomYMax = getZoomYMax(filteredPreppedData);
+
+  zoomYMax = getRoundedTickBoundary(zoomYMax || 0, yMaxZoom_Levels);
+
+  //reset zoom state
+  function handleZoomOut() {
+    setGlobalIsZoomed(false);
+    setFilteredPreppedData(preppedData);
+  }
+
+  useEffect(() => {
+    if (!zoomEnabled) return;
+
+    if (!globalIsZooming) {
+      const { x1, x2 } = globalZoomArea;
+
+      const dataInAreaRange = preppedData.filter(
+        (d) => d?.formattedDate >= x1 && d?.formattedDate <= x2
+      );
+      setFilteredPreppedData(dataInAreaRange);
+    }
+  }, [globalZoomArea, globalIsZooming, preppedData, zoomEnabled]);
+
   return (
     <div style={{ position: "relative", width: "100%", height: "100%" }}>
+      {zoomEnabled && globalIsZoomed && (
+        <div
+          className={
+            deltaView ? `absolute top-5 right-16 mr-3 z-10` : `absolute top-5 right-4 z-10`
```

Comment:
If you could separate out the shared classes from this ternary that would be great, putting them at the beginning and then using a template string for the conditional classes, keeps things neat and DRY